client/signup/steps/page-picker/shopping-cart-for-difm.tsx

```diff
@@ -195,6 +214,12 @@ export default function ShoppingCartForDIFM( {
 					<Total>
 						<div>{ translate( 'Total' ) }</div>
 						<div className="page-picker__value">{ total }*</div>
+						<RefundText>
+							<StyledBraveTickIcon />
+							{ translate( '%(days)d day money back guarantee', {
```

Comment:
I wonder if this might be better for translation and consistency purposes? https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20money-back%20guarantee&type=code

```suggestion
							{ translate( '%(days)d-day money-back guarantee', {
```

Reply:
Thanks for the suggestion!